<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post3dWindowNodeVectorArrowSettingDialog</class>
 <widget class="QDialog" name="Post3dWindowNodeVectorArrowSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arrow Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
     <item>
      <widget class="QGroupBox" name="commonSettinggroupBox">
       <property name="title">
        <string>Common Setting</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="physicalValueLabel">
            <property name="text">
             <string>Physical Value:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="physicalValueComboBox"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="lengthGroupBox">
          <property name="title">
           <string>Length</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="lengthAutoCheckBox">
               <property name="text">
                <string>Auto</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="stdValueLabel">
               <property name="text">
                <string>Standard value:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="stdValueSpinBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>6</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="stdValueUnitLabel">
               <property name="text">
                <string>[m/s]</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="legendLenLabel">
               <property name="text">
                <string>Length on screen:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="legendLengthSpinBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="legendLenUnitLabel">
               <property name="text">
                <string>[pixel]</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="minValueLabel">
               <property name="text">
                <string>Minimum value to draw:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="minValueSpinBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="minValueUnitLabel">
               <property name="text">
                <string>[m/s]</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>111</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="faceGroupBox">
       <property name="title">
        <string>Faces</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QListWidget" name="faceListWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Post3dWindowNodeGroupArrowFaceSettingWidget" name="faceSettingWidget" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="faceAddButton">
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="faceRemoveButton">
            <property name="text">
             <string>&amp;Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Post3dWindowNodeGroupArrowFaceSettingWidget</class>
   <extends>QWidget</extends>
   <header>post3dwindownodegrouparrowfacesettingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Post3dWindowNodeVectorArrowSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Post3dWindowNodeVectorArrowSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>stdValueSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>legendLengthSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minValueSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
